RoosterPlanner.Api/Models/PersonViewModel.cs

```diff
@@ -1,12 +1,51 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Linq;
+using Microsoft.Graph;
+using RoosterPlanner.Api.Models.Enums;
 
 namespace RoosterPlanner.Api.Models
 {
     public class PersonViewModel
     {
         public Guid Id { get; set; }
-        public string Name { get; set; }
+        public string FirstName { get; set; }
+        public string LastName { get; set; }
+        public string Email { get; set; }
+        public string StreetAddress { get; set; }
+        public string PostalCode { get; set; }
+        public string City { get; set; }
+        public string Country { get; set; }
+        public string DateOfBirth { get; set; }
+        public string PhoneNumber { get; set; }
+        public string UserRole { get; set; }
+        
+        public static PersonViewModel CreateVm(User user)
+        {
+            UserRole role = Enums.UserRole.None;
+            if (user.AdditionalData.ContainsKey($"extension_4e6dae7dd1c74eac85fefc6da42e7b61_UserRole")){
```

Comment:
Geen $-teken gebruiken voor string als er geen  { } in gebruikt worden.
De extension keys zou ik in een aparte .cs file opsommen als constanten...